src/main/play-doc/developer/ConductrSandbox.md

```diff
@@ -0,0 +1,110 @@
+```
+
+> Note that the ConductR sandbox will take a few seconds to become available and so any initial command that you send to it may not work immediately.
+
+Given the above you will then have a ConductR process running in the background (there will be an initial download cost for Docker to download the conductr/conductr-dev image from the public Docker registry).
+
+#### ConductR features
+
+ConductR gets shipped with features which can be optionally enabled during startup by specifying the `--withFeatures` option, e.g.:
+    
+```scala
+[my-app] sandbox run --withFeatures visualization logging
+[info] Running ConductR...
+[info] Running container cond-0 exposing 192.168.59.103:9000 192.168.59.103:9909...
+```
+
+The `visualization` feature provides a web interface to visualize the ConductR cluster together with deployed and running bundles. After enabling the feature, access it at http://{docker-host-ip}:9909. Replace `docker-host-ip` with your docker host ip address. For convience, the url of the visualizer app is displayed in the sbt session, e.g. http://192.168.59.103:9909.
+
+[[images/visualizer_simple.png]]
+
+The `logging` feature consolidates the logging output of ConductR itself and the bundles that it executes. To view the consolidated log messsages enable [sbt-conductr](https://github.com/sbt/sbt-conductr) and then run:
+
+```scala
+conduct logs conductr-elasticsearch
+```
+
+For more information refer to the [[Logging|Logging]] section.
+
+### Debugging ConductR
+
+It is possible to debug your application inside of the ConductR sandbox:
+
+1. Start ConductR sandbox in debug mode:
+
+    ```scala
+    sandbox debug
+    ```
+2. This exposes a debug port to docker and the ConductR container. By default the debug port is set to `5005` which is the default port for remote debugging in IntelliJ. If you are happy with the default setting skip this step. Otherwise specify another port in the `build.sbt`:
+
+    ```scala
+    SandboxKeys.debugPort := 5432
+    ```    
+3. Create the application bundle. The JVM argument `-jvm-debug $debugPort` is automatically added to the bundle configuration when using:
+
+    ```scala
+    bundle:dist
+    ```
+4. Load and run your bundle to the sandbox, e.g. by using [sbt-conductr](https://github.com/sbt/sbt-conductr):
+
+    ```scala
+    conduct load <HIT THE TAB KEY AND THEN RETURN>
+    conduct run my-app
+    ```
+
+You should be now able to remotely debug the bundle inside the ConductRs sandbox with your favorite IDE.
+
+**IntelliJ**
+
+1. In IntelliJ navigate to `Run` -> `Edit Configurations`
+2. Create a new `Remote` configuration with these options:
+    - Transport: Socket
+    - Debugger mode: Attach
+    - Host: Your docker host ip, e.g. 192.168.59.103
+    - Port: `debugPort` of your application, e.g. 5005
+
+    [[images/conductr_sandbox_debugging_intellij.png]]
+3. Save this configuration and start remote debugging with `Run` -> `Debug` and then select your configuration.
```

Comment:
If we're going to have a section for IntelliJ then I think we should have one for Eclipse... WDYT?